general_rules/Organization.tex

```diff
@@ -29,13 +29,13 @@ \subsection{Schedule}
 \label{rule:schedule}
 
 \begin{enumerate}
-	\item \textbf{Test Blocks:} For each test one Test Block is scheduled during which each (qualified) team gets at least two Test Slots assigned. With the Restaurant test being the exception where only one Test Slot is assigned.
-
-	\item \textbf{Test Slots:} During a Test Slot a team has the amount of time specified for the test to attempt to solve it. 
-
+	\item \textbf{Test Blocks:} For each test one Test Block is scheduled during which each (qualified) team gets at least two, maximum five Test Slots assigned. With the Restaurant test being the exception where only one Test Slot is assigned.
+	
+	\item \textbf{Test Slots:} During a Test Slot a team has one attempt to solve the test in the test's specified amount of time 
```

Comment:
There is a rule that allows teams to request a restart if not more than 50% of the time has passed yet. This conflicts with that but I dont think that rule is ever used, so I would opt to remove that instead.

Reply:
This rule has been removed as of #778